proto/comment/v1/comment.proto

```
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+package comments.v1;
```

Buf (1.55.1)

Files with package "comments.v1" must be within a directory "comments/v1" relative to root but were in directory "comment/v1".

(PACKAGE_DIRECTORY_MATCH)